site/pages/api/create-disruption.test.ts

```diff
@@ -265,14 +256,8 @@ describe("createDisruption", () => {
         disruptionData["publish-start-date"] = formatDate(publishStartDate, CD_DATE_FORMAT);
 
         const errors: ErrorInfo[] = [
-            {
-                id: "some-error-id",
-                errorMessage: "The Start date and time should be past the current date and time",
-            },
-            {
-                id: "some-error-id",
-                errorMessage: "The End date and time should be past the current date and time",
-            },
+            { id: "some-error-id", errorMessage: "The Start date and time should be past the current date and time" },
```

Comment:
Is this the current functionality? We should be allowing users to enter start dates that are in the past, confirmed with Steve.

Reply:
@alainajaffer I think we probably need to raise a bug rather than fix it in your PR, unless you feel its an easy change.

Reply:
this was only changed to make sure my ticket builds as we removed some of the disruption inputs. This isn't an api ticket and will need a ticket to fix the api